code/modules/admin/verbs/bluespacetech.dm

```diff
@@ -329,6 +329,14 @@
 		C.holder.original_mob = null
 	suicide()
 
+/mob/living/carbon/human/bst/verb/tgm()
```

Comment:
Why? BSTs are invulnerable in more ways than just GODMODE, just spawn a normal mob for testing stuff with damage.